src/engine/UIManager.cxx

```diff
@@ -1108,3 +1132,19 @@ void UIManager::changeFullScreenMode(UIElement *sender)
 
   MapFunctions::instance().refreshVisibleMap();
 }
+
+ImFont* ImGuiVars::buttonFontMainMenu = nullptr;
+ImFont* ImGuiVars::buttonFontDefault = nullptr;
+void ImGuiVars::init() {
+  json uiLayout;
+  auto& uiMgr = UIManager::instance();
+  uiMgr.loadSettings(uiLayout);
+  uiMgr.parseLayouts(uiLayout);
```

Comment:
I think we should extract UISettings class out from UIMgr.
If you agree, make a new issue for that one. Does not need to be part of this review change.

Reply:
sure, good idea